package.json

```diff
@@ -20,9 +20,11 @@
   "homepage": "https://github.com/adobe/aem-boilerplate#readme",
   "devDependencies": {
     "@babel/eslint-parser": "7.24.8",
-    "eslint": "8.57.0",
+    "@eslint/js": "^9.9.0",
+    "eslint": "^8.57.0",
     "eslint-config-airbnb-base": "15.0.0",
     "eslint-plugin-import": "2.29.1",
+    "globals": "^15.9.0",
```

Comment:
Where are you using globals? Always go with core javascript instead of additional packages.